api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -658,11 +658,9 @@ public ResponseEntity<WorkspaceBillingUsageResponse> getBillingUsage(
             .cost(freeTierBillingService.getWorkspaceFreeTierBillingUsage(workspace)));
   }
 
-  private WorkspaceUserRolesResponse shareWorkspace(
```

Comment:
before: a helper method common to 2 endpoints
now: directly implement the remaining endpoint